src/org/traccar/protocol/IdplProtocolDecoder.java

```diff
@@ -94,14 +94,14 @@ protected Object decode(
 
         position.set(Position.KEY_SATELLITES, parser.nextInt());
         position.set(Position.KEY_RSSI, parser.nextInt());
-        parser.next(); // vehicle status
+        position.set("vehicleStatus", parser.next()); // vehicle status
```

Comment:
Position.KEY_STATUS seems to be used throughout the code to represent the device status.  The IdplProtocol returns 5 types of status: vehicle, main power, AC, ignition, and output1.  If you feel that KEY_STATUS should be used, please confirm and I will change it.